FLEDGE_browser_bidding_and_auction_API.md

```diff
@@ -10,9 +10,9 @@ This document seeks to propose an API for web pages to perform FLEDGE auctions u
 
 #### Step 1: Get auction blob from browser
 
-To execute an on-server FLEDGE auction, sellers begin by calling `navigator.startServerAdAuction()`:
+To execute an on-server FLEDGE auction, sellers begin by calling `navigator.startServerAdAuction(): Promise<Uint8Array>`:
```

Comment:
Is using a colon the normal way to express the return type?

Reply:
I took inspiration from how you usually type function return values in TypeScript. We can probably just verbally describe the type though:

> by calling `navigator.startServerAdAuction()` which returns a `Promise<Uint8Array>`

Which do you prefer?

Reply:
I prefer the prose version as I think it aligns more with the rest of the doc

Reply:
Done, PTAL.